test/test_math_ops.cpp

```diff
@@ -389,6 +389,47 @@ TEST(MathOps, SquaredDifferenceBroadcasting) {
   opexecuter.RunTest();
 }
 
+TEST(MathOps, Abs1d) {
+  Scope root = Scope::NewRootScope();
+  int dim1 = 1;
+
+  Tensor A(DT_FLOAT, TensorShape({dim1}));
```

Comment:
By abs1d, we mean a abs test on a rank1 tensor (or a vector).
If my understanding is right this is how you might define a vector of length 5
```
TensorShape({dim1})
```

The fact that there is only 1 number in the braced list means it is rank 1. however that number itself can be non-one.

In this test you have dim1 set to 1. So '1' becomes a bit overloaded (rank is 1 and dim is also 1). Also we are testing a degenerate/trivial vector since it is basically a single number

TL;DR: set dim1 to some integer > 1

Reply:
Changed.